packages/emscripten/build.sh

```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE=https://emscripten.org/
+TERMUX_PKG_DESCRIPTION="Emscripten is a complete compiler toolchain to WebAssembly"
+TERMUX_PKG_LICENSE="MIT"
+TERMUX_PKG_VERSION=2.0.9
+TERMUX_PKG_SRCURL=https://github.com/emscripten-core/emscripten/archive/${TERMUX_PKG_VERSION}.tar.gz
+TERMUX_PKG_SHA256=9e54b33da5025270221b38fdc603191b8f22afa038b72c6dcfaa6423888746f5
+TERMUX_PKG_DEPENDS="python, llvm, nodejs, binaryen"
+
+termux_step_post_get_source() { 
+	export EM_BINARYEN_ROOT=$TERMUX_PREFIX/usr/bin
```

Comment:
So I need to set the full path of binaryen's binaries. 
apt install this one into /usr/bin, so need  `$TERMUX_PREFIX/usr/bin`

Reply:
`$TERMUX_PREFIX/usr/bin` = `/data/data/com.termux/files/usr/usr/bin`, you probably do not want /usr/usr

Reply:
Then binaryen has to be fixed.

/usr in Termux context is not available, only $TERMUX_PREFIX which stands for / and /usr at one time.

Reply:
update to $TERMUX_PREFIX/bin

Reply:
no need to fix binaryen, install [path ](https://github.com/termux/termux-packages/blob/master/packages/binaryen/build.sh) is right